specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -10912,6 +10915,26 @@
       },
       "title": "The execution constraints for a job."
     },
+    "TaskContainerSettings": {
+      "properties": {
+        "containerRunOptions": {
+          "type": "string",
+          "title": "Additional options to the container run command.",
+          "description": "These additional options are supplied as arguments to the \"docker create\" command, in addition to those controlled by the Batch Service."
+        },
+        "imageName": {
+          "type": "string",
+          "title": "The container image under which the task will run."
```

Comment:
I find this wording a bit awkward but not sure how to improve it without getting quite verbose, e.g. "The image to use to create the container in which the task will run."  I know that's nasty but maybe we can bat some wording around.

Reply:
Yes I don't love the "under which" either.

I do like yours more than what I have currently...
Maybe we could clean it up a bit by saying "The container image in which the task will run."

It's less technically correct because the task doesn't run in the image it runs in the container created from the image... but for those familiar with Docker it might be clearer than yours (not sure)?

Reply:
My thought was "The container image in which the task will run" is clear (even if it's not technically the way it works).